src/mpack/conv.c

```diff
@@ -327,8 +327,6 @@ static mpack_value_t mpack_pack_ieee754(double v, unsigned mantbits,
   double mant;
 
   if (v == 0) {
-    rv.lo = 0;
```

Comment:
does it still pass if we remove the initializer above instead?

Reply:
You mean pass as in the warning will disappear? Can't say for certain since the PVS list updates the next day, but I think that also should also work if you prefer it that way instead.

Reply:
Added a fixup with what I think is the solution you wanted.

Reply:
I meant all the CI madness I guess, so let's merge my suggestion and see what happens

Reply:
The CI passes now.

Reply:
> Can't say for certain since the PVS list updates the next day,

You can run PVS locally, just like CI does. :)

Reply:
The pvscheck script is borked.

Reply:
It's a cmake error which means it's impossible to solve.

Reply:
Can take a look at how the CI calls it, maybe there's a flag I missed.

Reply:
OK, got it working by following the CI. Thanks for the tip!

Reply:
should be upstreamed. see also https://github.com/libmpack/libmpack/issues/3

wonder if we can get commit access @tarruda

Reply:
Removing the mpack changes as to not block this PR.